### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by andrey-star**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (stofee) [inline LICENSE:189]: Copyright 2020 Google LLC.

Comment (andrey-star) [reply LICENSE]: Done

**Comment by andrey-star** [comment]
I took a look at other licensed open-source products (https://github.com/angular/angular-cli), they have licenses on the sources, but not config files such as `.gitignore` or `package.json`

File: step-release-vis/src/app/components/environment/environment_test.ts [hunk 2]
```diff
@@ -10,7 +15,6 @@ import {CandidateServiceStub} from '../../../testing/CandidateServiceStub';
 import {CandidateService} from '../../services/candidateService';
 import {DebugElement, SimpleChange} from '@angular/core';
 import {TooltipComponent} from '../tooltip/tooltip';
-import {shouldBeautify} from '@angular-devkit/build-angular/src/utils/environment-options';
```
Comment (stofee) [inline step-release-vis/src/app/components/environment/environment_test.ts:13]: why to remove this? pls do separate pr if intended

File: step-release-vis/src/environments/environment.prod.ts [hunk 3]
```diff
@@ -1,3 +1,8 @@
+/*
+ * Copyright 2020 Google LLC.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
 export const environment = {
-  production: true
```
Comment (stofee) [inline step-release-vis/src/environments/environment.prod.ts:2]: pls do a separate pr for nit fixes

File: step-release-vis/src/main.ts [hunk 4]
```diff
@@ -1,12 +1,18 @@
...
+
+import {AppModule} from './app/app.module';
+import {environment} from './environments/environment';
 
 if (environment.production) {
   enableProdMode();
 }
 
-platformBrowserDynamic().bootstrapModule(AppModule)
+platformBrowserDynamic()
```
Comment (stofee) [inline step-release-vis/src/main.ts:16]: pls separate format hanges

File: step-release-vis/src/polyfills.ts [hunk 5]
```diff
@@ -55,8 +60,7 @@
 /***************************************************************************************************
  * Zone JS is required by default for Angular itself.
  */
-import 'zone.js/dist/zone';  // Included with Angular CLI.
-
+import 'zone.js/dist/zone'; // Included with Angular CLI.
```
Comment (stofee) [inline step-release-vis/src/polyfills.ts:63]: not in this pr

Comment (andrey-star) [reply step-release-vis/src/polyfills.ts:63]: sorry, the file was reformatted on save

File: step-release-vis/src/polyfills.ts [hunk 6]
```diff
@@ -57,7 +62,6 @@
  */
 import 'zone.js/dist/zone';  // Included with Angular CLI.
 
-
```
Comment (stofee) [inline step-release-vis/src/polyfills.ts]: readd emtpy line here
